pumps/mongo_aggregate.go

```diff
@@ -170,12 +177,32 @@ func (m *MongoAggregatePump) Init(config interface{}) error {
 	if m.dbConf.ThresholdLenTagList == 0 {
 		m.dbConf.ThresholdLenTagList = THRESHOLD_LEN_TAG_LIST
 	}
+	// if StoreAnalyticsPerMinute is set to true, the aggregation time will be set to 1.
+	// if not, the aggregation time will be set to the value of the field AggregationTime.
+	// if there is no value for AggregationTime, it will be set to 60.
+	if m.dbConf.StoreAnalyticsPerMinute {
```

Comment:
@mativm02 can we encapsulate this into a separated function and add tests? it's going to be easier to test.

Reply:
We can encapsulate it into a function (sending a pointer to MongoAggregatePump as a parameter) or into a *MongoAggregatePump method. Which one would be the best way?